macosx/Base.lproj/GroupRules.xib

```diff
@@ -6,7 +6,7 @@
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <objects>
-        <customObject id="-2" userLabel="File's Owner" customClass="GroupsPrefsController">
+        <customObject id="-2" userLabel="File's Owner" customClass="GroupsPrefsController" customModule="Transmission" customModuleProvider="target">
```

Comment:
Is this something that is required by Swift to work, or some changes that Xcode makes automatically?
If it's the latter let's revert(or extract to a separate PR) all changes in xibs to make this PR a bit smaller and easier to review

Reply:
It's a required change for Swift to work, and I did the change manually